packages/component-library/src/components/plmg-dropdown-item/plmg-dropdown-item.scss

```diff
@@ -0,0 +1,72 @@
+@use '~@ducky/plumage-tokens/dist/scss/default' as tokens;
+
+// Component style
+// -------------------------------------------------
+.plmg-dropdown-item {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: tokens.$plmg-spacing-x0-75 tokens.$plmg-spacing-x1;
+  cursor: pointer;
+  border-radius: tokens.$plmg-border-radius-s;
+  background: tokens.$plmg-color-background-neutral;
+  color: tokens.$plmg-color-text-neutral-weak;
+  max-width: 312px;
+
+  // remove default link styles
+  text-decoration: none;
+  border: none;
+  list-style: none;
+
+  // common typography
+  font-family: tokens.$plmg-font-family-sans;
+  font-style: normal;
+  font-weight: tokens.$plmg-font-weight-regular;
+  line-height: tokens.$plmg-font-lineheight-loose;
+  font-size: tokens.$plmg-font-size-x0-875;
+
+  .plmg-dropdown-item-content {
+    display: block;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
+  &.plmg-dropdown-item-active {
```

Comment:
this class is not used in the code

Reply:
this raises a question: Do we need a way to set the currently active item?
If the dropdown menu is used to navigate to a page, when I'm on the page I clicked, the item should be "active"?

Reply:
Just an active prop on the item? Or add in a whole focus active management system into the parent ? I'm trying to avoid overriding the default states (beyond styling them). I tried this with arrow key navigation and found that once you start controlling one state you have to control them all.